### DISCUSSION TITLE
Validar tipos de datos en variables de Blogger

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#39] [Announcements]
# Validar tipos de datos en variables de Blogger

Aprende a establecer tipos de datos de los valores ingresados en las variables de Blogger

https://www.zkreations.com/2024/07/validar-tipos-de-datos-variables-blogger.html

**Comment by ghost** [comment]
Dude, can you turn on the RSS feature of your blog?

**Reply by danieIabel** [reply]
Good evening, my friend.

The feed of my blog is activated. Have you had any issues with the feed?

**Reply by ghost** [reply]
I can't see this blog's RSS in my RSS reader.

**Reply by danieIabel** [reply]
I tried using the blog's XML file: https://www.zkreations.com/rss.xml
